clang/test/Driver/sycl-offload-arch-intel-cpu.cpp

```diff
@@ -0,0 +1,78 @@
+/// Tests the behaviors of using -fsycl --offload-new-driver 
+//  --offload-arch=<intel-cpu-values>.
+
+// SYCL AOT compilation to Intel CPUs using --offload-arch
+
+// RUN: %clangxx -### --offload-new-driver -fsycl --offload-arch=broadwell %s 2>&1 | \
```

Comment:
Do we need the --offload-new-driver flag here? I thought it was turned on by default in the upstream SYCL compilation flow.

Thanks